ketcher-autotests/tests/utils/common/waitForSpinnerFinishedWork/waitForSpinnerFinishedWork.ts

```diff
@@ -0,0 +1,11 @@
+import { Page } from '@playwright/test';
```

Comment:
Good job!

I have a proposal, may be we can rename `waitForSpinnerFinishedWork` to `waitForLoading`? 
And we already have the same function. Do we need. to create a separate one? May be we can modify existing one?

Additional proposal is to create separate folder like `tests/utils/.../loaders` and put the functions like these there?